.gitignore

```diff
@@ -53,3 +52,6 @@ tmp/
 testing
 nix-testing
 test
+
+# Ignore nix build
+result
```

Comment:
All changes in this file look to be unrelated and possibly related to your other pr #180